src/bindgen/builder.rs

```diff
@@ -293,6 +296,20 @@ impl Builder {
         self
     }
 
+    /// Track the source (file) of the config to support the `--depfile` option
+    #[allow(unused)]
+    pub(crate) fn with_config_and_src(
+        mut self,
+        config: Config,
+        config_src: Option<path::PathBuf>,
```

Comment:
Any reason we could not just store the `src_path: Option<PathBuf>` in the `Config`? That seems like it'd be cleaner?

Reply:
The `config_src` should contain the path to the config file (if it exists). 
The `Config` itself is parsed from the config file. It would be strange if the user could set a path in the config file, which should point to the config file itself.

Reply:
Finished the change, this is indeed much cleaner.